docs/classic-vs-vt.md

```diff
@@ -0,0 +1,81 @@
+
+## Comparisons
+
+### Cross-Platform Support
+
+As every platform in the world, except Windows, natively spoke virtual terminal sequences, both terminal applications and command-line utilities have been easily portable between versions and variations of those operating systems. By contrast, Windows APIs are only supported on Windows. An extensive adapter or translation library has to be written between Windows and VT or vice versa when attempting to port command-line utilities from one platform or another.
+
+### Remote Access
+
+Virtual Terminal Sequences hold a major advantage for remote access as they require no additional work to transport or perform remote procedure calls over what is required to set up a standard remote command-line connection. Simply connecting an outbound and an inbound transport channel (or a single bidirectional channel) over a pipe, socket, file, serial port, or any other device is sufficient to completely carry all information necessary required for an application speaking these sequences to a remote host. On the contrary, the Windows Console APIs have only been accessible on the local machine and all efforts to remote them would require building an entire remote calling and transport interface layer beyond just a simple channel.
+
+### Separation of Concerns
+
+Some of the Windows Console APIs provide low-level access to the input and output buffers or convenience functions for interactive command-lines like aliases and command history programmed within the console subsystem and host environment, instead of within the command-line client application itself. This is in contrast to other platforms where both the memory of the current state of the application and convenience functionality is the responsibility of the command-line utility or shell itself.
+
+While the Windows way of handling this responsibility in the console host and API makes it quicker and easier to write a command-line application with these features and without responsibility of remembering drawing state or handling editing convenience features, it makes it nearly impossible to connect those activities remotely across platforms, versions, or scenarios due to variations in implementations and availability. It also makes the final interactive experience of these Windows command-line applications completely dependent on the console host's implementation, priorities, and release cycle.

```

Comment:
Might be a good place to mention advanced line editing features like syntax highlighting, text selection, etc?